workspaces/graph-lib/src/endpoints-graph/index.ts

```diff
@@ -631,7 +669,8 @@ export class GraphQueries {
   }
 
   //@TODO move away from null here
-  wrapList(type: NodeType | null, nodes: Node[]): NodeListWrapper {
+  // TODO figure out how to make this generic
+  wrapList(type: NodeType | null, nodes: Node[]): NodeListWrapper<NodeWrapper> {
```

Comment:
I couldn't figure out how or why wrapList and wrap weren't playing nice with generics. There's probably a way to do this, but it probably requires some type juggling. The impact of this is we just need to cast `as NodeListWrapper<NodeTypeToNodeWrapper<T>>` when we call wrapList (which is only used internally in this class) - which is much better than having the consumers have to cast things